### ISSUE TITLE
librispeech rnnt tlg解码报错

### ISSUE THREAD

**Issue by manbaaaa** [wenet-e2e/wenet#1620]
fsttablecompose data/lang_test/T.fst data/lang_test/LG.fst
Composing decoding graph TLG.fst succeeded
./tools/decode.sh: line 66: 389715 Aborted                 (core dumped) decoder_main --rescoring_weight $rescoring_weight --ctc_weight $ctc_weight --reverse_weight $reverse_weight --chunk_size $chunk_size --wav_scp ${dir}/split${nj}/wav.${n}.scp --model_path $model_file --unit_path $unit_file $wfst_decode_opts --result ${dir}/split${nj}/${n}.text &> ${dir}/split${nj}/${n}.log
./tools/decode.sh: line 66: 389711 Aborted                 (core dumped) decoder_main --rescoring_weight $rescoring_weight --ctc_weight $ctc_weight --reverse_weight $reverse_weight --chunk_size $chunk_size --wav_scp ${dir}/split${nj}/wav.${n}.scp --model_path $model_file --unit_path $unit_file $wfst_decode_opts --result ${dir}/split${nj}/${n}.text &> ${dir}/split${nj}/${n}.log
./tools/decode.sh: line 66: 389712 Aborted                 (core dumped) decoder_main --rescoring_weight $rescoring_weight --ctc_weight $ctc_weight --reverse_weight $reverse_weight --chunk_size $chunk_size --wav_scp ${dir}/split${nj}/wav.${n}.scp --model_path $model_file --unit_path $unit_file $wfst_decode_opts --result ${dir}/split${nj}/${n}.text &> ${dir}/split${nj}/${n}.log
./tools/decode.sh: line 66: 389716 Aborted                 (core dumped) decoder_main --rescoring_weight $rescoring_weight --ctc_weight $ctc_weight --reverse_weight $reverse_weight --chunk_size $chunk_size --wav_scp ${dir}/split${nj}/wav.${n}.scp --model_path $model_file --unit_path $unit_file $wfst_decode_opts --result ${dir}/split${nj}/${n}.text &> ${dir}/split${nj}/${n}.log
./tools/decode.sh: line 66: 389718 Aborted                 (core dumped) decoder_main --rescoring_weight $rescoring_weight --ctc_weight $ctc_weight --reverse_weight $reverse_weight --chunk_size $chunk_size --wav_scp ${dir}/split${nj}/wav.${n}.scp --model_path $model_file --unit_path $unit_file $wfst_decode_opts --result ${dir}/split${nj}/${n}.text &> ${dir}/split${nj}/${n}.log
./tools/decode.sh: line 66: 389719 Aborted                 (core dumped) decoder_main --rescoring_weight $rescoring_weight --ctc_weight $ctc_weight --reverse_weight $reverse_weight --chunk_size $chunk_size --wav_scp ${dir}/split${nj}/wav.${n}.scp --model_path $model_file --unit_path $unit_file $wfst_decode_opts --result ${dir}/split${nj}/${n}.text &> ${dir}/split${nj}/${n}.log
cat: exp/conformer_transducer/lm_with_runtime_test_clean/split6/1.text: No such file or directory

**Comment by manbaaaa** [comment]
log中的信息

I1217 07:30:44.030885 389711 params.h:147] Reading torch model exp/conformer_transducer/final.zip
I1217 07:30:44.059149 389711 torch_asr_model.cc:34] Num intra-op threads: 1
terminate called after throwing an instance of 'torch::jit::ErrorReport'
  what():
Unknown builtin op: torchaudio::rnnt_loss.
Could not find any similar ops to torchaudio::rnnt_loss. This op may not exist or may not be currently supported in TorchScript.
:
  File "code/__torch__/torchaudio/functional/functional.py", line 20
  else:
    blank0 = blank
  costs, _3 = ops.torchaudio.rnnt_loss(logits, targets, logit_lengths, target_lengths, blank0, clamp)
              ~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
  if torch.eq(reduction, "mean"):
    _4, _5 = True, torch.mean(costs)
'rnnt_loss' is being compiled since it was called from 'Transducer.forward'
Serialized   File "code/__torch__/wenet/transducer/transducer.py", line 28
    text_lengths: Tensor) -> Dict[str, Optional[Tensor]]:
    _0 = __torch__.wenet.utils.common.add_blank
    _1 = __torch__.torchaudio.functional.functional.rnnt_loss
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    _2 = torch.eq(torch.dim(text_lengths), 1)
    if _2:

**Comment by qipotianMFXT** [comment]
Hello, may I ask if you have solved this problem?
